src/tables.ts

```diff
@@ -156,6 +156,7 @@ const validNoteTypes: Record<string, { name: string }> = {
   m: { name: 'muted' },
   s: { name: 'slash' },
   g: { name: 'ghost' },
+  p: { name: 'parenthesized' },
```

Comment:
You can remove this (and the code below.)

Reply:
done